packages/create-graphback/src/init/components.ts

```diff
@@ -32,7 +32,7 @@ async function chooseTemplate(filter: string = ''): Promise<Template> {
     {
       type:'list',
       name:'typeOfTemplates',
-      message:'Choose between community or officail Graphback templates',
+      message:'Choose between community or official Graphback templates',
```

Comment:
Would it be possible to print them separately? Too much ask?

Reply:
Separately as in no choosing and just all of them listed with some gap (or a line saying that 'following are the community templates') in between?